core-blocks/text-columns/index.js

```diff
@@ -64,6 +68,12 @@ export const settings = {
 	edit: ( ( { attributes, setAttributes, className } ) => {
 		const { width, content, columns } = attributes;
 
+		deprecated( 'The Text Columns block', {
+			version: 'a future version',
```

Comment:
`version` is an optional param, so we could just omit it entirely instead of being vague here.

Reply:
Oh—we should update `deprecated.md`.